pages/InFeed.js

```diff
@@ -0,0 +1,126 @@
+import { Text, View, ImageBackground, ScrollView, StyleSheet, Button, Image } from 'react-native'
+import { Card } from 'react-native-elements'
+import { LinearGradient } from 'expo-linear-gradient'
+import * as React from 'react'
+import InFeedItem from '../components/InFeedItem'
+
+export default function InFeed() {
+  const styles = StyleSheet.create({
+    image: {
+      flex: 1,
+      justifyContent: 'center'
+    },
+    title: {
+      color: 'white',
+      fontSize: 30,
+      lineHeight: 84,
+      textAlign: 'center'
+    },
+    subtitle: {
+      color: 'white',
+      fontSize: 20,
+      lineHeight: 84,
+      textAlign: 'center'
+    },
+    titleBody: {
+      color: 'black',
+      fontSize: 25,
+      fontWeight: 'bold',
+      textAlign: 'justify'
+    },
+    subtitleBody: {
+      color: 'black',
+      fontSize: 20,
+      textAlign: 'justify'
+    },
+    fakeArticle: {
+      height: 10,
+      width: '100%',
+      backgroundColor: '#cfcfcf',
+      opacity: 0.32,
+      borderRadius: 10,
+      marginTop: 10
+    },
+    background: {
+      position: 'absolute',
+      left: 0,
+      right: 0,
+      top: 0,
+      height: 170
+    }
+  })
+
+  return (
+    <ScrollView style={{ flexGrow: 1 }}>
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'center',
+          alignItems: 'stretch'
+        }}
```

Comment:
This comment is valid for almost all your react components:

You are creating Stylesheets to style your components but you are also using `style` props, why not using only `StyleSheet.create` to style your components ? 

This will centralize your style configuration in one place and not have to scroll down and up to see how the components is stylized

Reply:
Yeah sure it's clearer, I've put all styles in one file named styles.js let me know if it's better 😄